.github/workflows/conformance.yml

```diff
@@ -24,46 +24,64 @@ jobs:
         go-version: '1.16'
 
     - name: Run HTTP conformance tests
-      uses: GoogleCloudPlatform/functions-framework-conformance/action@v1.1.0
+      uses: GoogleCloudPlatform/functions-framework-conformance/action@v1.6.0
       with:
-        version: 'v1.1.0'
+        version: 'v1.6.0'
         functionType: 'http'
         useBuildpacks: false
         validateMapping: false
         cmd: "'functions-framework --source tests/conformance/main.py --target write_http --signature-type http'"
 
     - name: Run event conformance tests
-      uses: GoogleCloudPlatform/functions-framework-conformance/action@v1.1.0
+      uses: GoogleCloudPlatform/functions-framework-conformance/action@v1.6.0
       with:
-        version: 'v1.1.0'
+        version: 'v1.6.0'
         functionType: 'legacyevent'
         useBuildpacks: false
         validateMapping: true
         cmd: "'functions-framework --source tests/conformance/main.py --target write_legacy_event --signature-type event'"
 
     - name: Run CloudEvents conformance tests
-      uses: GoogleCloudPlatform/functions-framework-conformance/action@v1.1.0
+      uses: GoogleCloudPlatform/functions-framework-conformance/action@v1.6.0
       with:
-        version: 'v1.1.0'
+        version: 'v1.6.0'
         functionType: 'cloudevent'
         useBuildpacks: false
         validateMapping: true
         cmd: "'functions-framework --source tests/conformance/main.py --target write_cloud_event --signature-type cloudevent'"
 
     - name: Run HTTP conformance tests declarative
-      uses: GoogleCloudPlatform/functions-framework-conformance/action@v1.1.0
+      uses: GoogleCloudPlatform/functions-framework-conformance/action@v1.6.0
       with:
-        version: 'v1.1.0'
+        version: 'v1.6.0'
         functionType: 'http'
         useBuildpacks: false
         validateMapping: false
         cmd: "'functions-framework --source tests/conformance/main.py --target write_http_declarative'"
 
     - name: Run CloudEvents conformance tests declarative
-      uses: GoogleCloudPlatform/functions-framework-conformance/action@v1.1.0
+      uses: GoogleCloudPlatform/functions-framework-conformance/action@v1.6.0
       with:
-        version: 'v1.1.0'
+        version: 'v1.6.0'
         functionType: 'cloudevent'
         useBuildpacks: false
         validateMapping: true
         cmd: "'functions-framework --source tests/conformance/main.py --target write_cloud_event_declarative'"
+
+    - name: Run HTTP concurrency tests declarative
+      uses: GoogleCloudPlatform/functions-framework-conformance/action@v1.6.0
+      with:
+        version: 'v1.6.0'
+        functionType: 'http'
+        useBuildpacks: false
+        validateConcurrency: true
+        cmd: "'functions-framework --source tests/conformance/main.py --target write_http_declarative_concurrent'"
+
+    - name: Run CloudEvents concurrency tests
```

Comment:
Eh, let's just run it on HTTP functions? Seems unnecessary to run it on both since it's not a cloudevent vs http feature.

Reply:
Sure thing